apps/lib/signer/cgo_flags.go

```diff
@@ -1,7 +1,7 @@
 package signer
 
 /*
-#cgo LDFLAGS: -L./rust/stork/target/release -lstork
+#cgo LDFLAGS: -L./rust/stork/target -L./../../../../.lib -lstork
```

Comment:
needed to do this to run locally after building with `make rust` - could be doing something wrong though

Reply:
What are you doing to run locally? This shouldnt be necessary